### PR TITLE
fix shader for gvr-outline sample

### PR DESCRIPTION
**Pull request by roshanch**
It doesn't like gl_FragColor.

GearVRf-DCO-1.0-Signed-off-by: Roshan Chaudhari roshan.c@samsung.com

### CODE REVIEW

File: gvr-outline/app/src/main/java/org/gearvrf/gvroutline/OutlineShader.java [hunk 1]
```diff
@@ -37,10 +37,10 @@
 
     private static final String FRAGMENT_SHADER =
               "precision mediump float;\n"
-            + "out vec4 gl_FragColor;\n"
+            + "out vec4 outColor;\n"
```
Comment (liaxim) [inline gvr-outline/app/src/main/java/org/gearvrf/gvroutline/OutlineShader.java]: For completeness should the shader also specify version 300 explicitly?

Comment (roshanch) [inline gvr-outline/app/src/main/java/org/gearvrf/gvroutline/OutlineShader.java]: We are adding that in shaderTemplate.java

**Comment by liaxim** [comment]
Will merge in the afternoon if there are no further comments.

**Comment by thomasflynn** [comment]
since the PR went up after the build team enabled the pre-merge build polling, i'm assuming we should a pass/fail from the build system sometime soon?  (since it polls every hour)?

**Comment by liaxim** [comment]
Ok, will wait for the build result.

**Comment by thomasflynn** [comment]
just talked to Shreeti.  it doesn't work the way one would hope.  because IT won't allow proper communication with github (*frustration scream*), what she set up will only trigger a build _after_ we click 'merge pull request'.  Then, we'll know _after the fact_ if the build got broken.  It's better than nothing, but is still close to nothing :(.

**Comment by liaxim** [comment]
Texture is missing. Will investigate tomorrow.

**Comment by roshanch** [comment]
There was issue in batching code, we set material to null in renderCamera() but batching code was setting it to renderpass(0) material, and we don't update material from render-pass if its not null in non-batch mode. PR #775 is updated.

**Comment by liaxim** [comment]
Verified. Will merge after lunch.

**Pull request merged**